projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/linked-cases/components/linked-cases-table/linked-cases-from-table.component.scss

```diff
@@ -0,0 +1,8 @@
+.case-table-column {
+  min-width: 20%;
+  max-width: 20%;
+}
+
+.heading-h2 {
+	margin-bottom: 0;
```

Comment:
@johnbenjamin-hmcts - seems to be extra whitespace here

Reply:
@Daniel-Lam I have formatted the code.